scripts/Muon/GUI/Common/calculate_pair_and_group.py

```diff
@@ -29,12 +30,12 @@ def calculate_pair_data(pair: MuonPair, forward_group: str, backward_group: str,
 
 
 def estimate_group_asymmetry_data(context, group, run, rebin, workspace_name, unormalised_workspace_name, periods):
-    processed_data = get_pre_process_workspace_name(run, context.data_context.instrument)
-
-    params = _get_MuonGroupingAsymmetry_parameters(context, group, run, periods)
-    params["InputWorkspace"] = processed_data
-    group_asymmetry, group_asymmetry_unnorm = algorithm_utils.run_MuonGroupingAsymmetry(params, workspace_name,
-                                                                                        unormalised_workspace_name)
+    params = _get_EstimateMuonAsymmetryFromCounts_parameters(context, group, run, periods)
+    params["InputWorkspace"] = get_group_data_workspace_name(
+        context, group.name,get_run_numbers_as_string_from_workspace_name(
+            workspace_name, context.data_context.instrument), "", rebin)
```

Comment:
what is the value of `workspace_name` at this point?

Reply:
it is of the format ``MUSR62260; Group; top; Asymmetry; MA``, we need ``MUSR62260; Group; top; Counts; MA``?

Reply:
you could just replace the asymmetry with counts in the name. Alternatively you might be able to just cast the run as a string

Reply:
Run is actually a list, and so you would have to cast one of the items in the list. This is fine for single runs but for co-add the run list has multiple entries, so to get the full run number you need to call that function. I guess that could be avoided by just doing something like 
```
params["InputWorkspace"] = workspace_name.replace("Asymmetry", "Counts")
```

Reply:
yes it can, its not an ideal solution, but its an easy one. Just add it to the list https://github.com/mantidproject/mantid/issues/31997